diff --git a/src/app/state/settings.ts b/src/app/state/settings.ts
index 9d97919..b4376e2 100644
--- a/src/app/state/settings.ts
+++ b/src/app/state/settings.ts
@@ -35,6 +35,9 @@ export interface Settings {
   isNotificationSounds: boolean;
 
   developerTools: boolean;
+
+  enableBottomGesture: boolean;
+  displayBottomGestureHint: boolean;
 }
 
 const defaultSettings: Settings = {
@@ -61,9 +64,12 @@ const defaultSettings: Settings = {
   showHiddenEvents: false,
 
   showNotifications: true,
   isNotificationSounds: false,
 
   developerTools: false,
+
+  enableBottomGesture: false,
+  displayBottomGestureHint: true,
 };
 
 export const getSettings = () => {
